### PR TITLE
Ndcc slides

### PR DESCRIPTION
**Pull request by trey-stafford**
Presentation I gave at the 2024-03-26 NSIDC developers roundtable. I plan to adapt the presentation for use at the NSIDC science team meeting in April.

### CODE REVIEW

File: presentations/ndcc-overview.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: "National Discovery Cloud for Climate workshop overview"
+date: "2024-03-26"
+format: revealjs
+
+author: "Trey Stafford"
```
Comment (mfisher87) [inline presentations/ndcc-overview.md:6]: Don't forget your ORCiD! The decision records have an example of how to format that.

File: presentations/ndcc-overview.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+software, and networking resources, democratizing access to a
+cyberinfrastructure ecosystem that is increasingly necessary to further
+climate-related science and engineering”. The NDC-C builds on many existing
+efforts supported by the CISE and GEO directorates across the entire science
+vertical from hardware to foundational infrastructure to software services to
+core climate research to community activities." -
+https://www.nationaldiscoverycloud.org/home
+
+CISE == Directorate for Computer and Information Science and Engineering
+GEO == Geosciences Directorate
```
Comment (mfisher87) [inline presentations/ndcc-overview.md:35]: Can these be moved to [footnotes](https://quarto.org/docs/presentations/revealjs/#asides-footnotes) on the slide?

**Review by mfisher87 — APPROVED** [review]
Sick presentation today :) :100:

File: presentations/ndcc-overview.md (hunk 2 above)
Comment (trey-stafford) [reply presentations/ndcc-overview.md:35]: Good suggestion! This is part of the speaker notes ( I don't refer to CISE or GEO in the slide content), but I may end up referring to these later and that would be a good way to put that info on the screen!

File: presentations/ndcc-overview.md (hunk 1 above)
Comment (trey-stafford) [reply presentations/ndcc-overview.md]: Nice suggestion, thanks!

**Pull request merged**